README.md

```diff
@@ -18,17 +18,18 @@ ctcf_orientation.py -l <loopfile> -m <motiffile> -o <outputfile>
 
 - -l; loop file: Pairedbed file of loops with the format "chromosome1 anchor1_start anchor1_end chromosome2 anchor2_start anchor2_end"
 - -m; motif file: CTCF motif file in the format "chromosome start end name strength orientation pvalue qvalue sequence", although only "chromosome start end" will be used, with the other columns optional. 
-- -o; output file: Desired file name for the outputted piechart with extensions available in matplotlib. 
+- -o; output file: Desired file name for the output piechart of the breakdown of Loop overlaps; will also output another file with the form outputfile_breakdown.extension to display percentage of loops that overlap CTCF coordinates.  
```

Comment:
Change "of the breakdown of Loop overlaps" to "of the breakdown of Loop CTCF orientation"